programs/svm-spoke/src/instructions/fill.rs

```diff
@@ -179,27 +161,18 @@ pub struct CloseFillPda<'info> {
         bump,
         close = signer, // TODO: check if this is correct party to receive refund.
         // Make sure caller provided relay_hash used in PDA seeds is valid.
-        constraint = is_relay_hash_valid(&relay_hash, &relay_data, &state) @ CustomError::InvalidRelayHash
+        constraint = is_relay_hash_valid(&relay_hash, &relay_data, &state) @ SvmError::InvalidRelayHash
     )]
     pub fill_status: Account<'info, FillStatusAccount>,
 }
 
-pub fn close_fill_pda(
-    ctx: Context<CloseFillPda>,
-    _relay_hash: [u8; 32], // TODO: check if we can use underscore in functions while in context macro leave as is?
-    relay_data: V3RelayData
-) -> Result<()> {
+pub fn close_fill_pda(ctx: Context<CloseFillPda>, _: [u8; 32], relay_data: V3RelayData) -> Result<()> {
     let state = &ctx.accounts.state;
     let current_time = get_current_time(state)?;
 
-    // Check if the fill status is filled
```

Comment:
note that I changed the requires here! I think this check is redundant. the ONLY thing we care about is if the deposit is expired, not if it was filled (or not) to let us close the PDA. LMK if you'll agree.